quickstarts/kubernetes-prometheus/config.yml

```diff
@@ -0,0 +1,44 @@
+slug: kubernetes-prometheus
+title: Kubernetes (Prometheus)
+description: |
+  ## What is Kubernetes?
+  Kubernetes is an open-source system for automating deployment, scaling, and, management of containerized applications. The New Relic Kubernetes (Prometheus) monitoring quickstart utilizes Prometheus to give you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
+  
+  ## What is Prometheus? 
+  Prometheus is an open-source monitoring and alerting toolkit. 
+
+  ### Kubernetes (Prometheus) quickstart highlights
+  The New Relic Kubernetes quickstart uses dashboards to proactively monitor your metrics, like:
```

Comment:
I would say the New Relic Prometheus quickstart for Kubernetes, and I would mention that we also have a Kubernetes integration and link that as well